scripts/mysqld_safe.sh

```diff
@@ -215,10 +215,6 @@ eval_log_error () {
 
   #echo "Running mysqld: [$cmd]"
   eval "$cmd"
-  ret=$?
```

Comment:
What's the status of BLD-807 after this revert?

Reply:
Created a new BLD-893 and marked as a release blocker